### DISCUSSION TITLE
OS Fingerprinting

### DISCUSSION THREAD

**Discussion by issmonitor** [Security-Onion-Solutions/securityonion#3874] [Unsupported Versions] [answered]
Has anyone played around with OS fingerprinting in SO?  Have been wanting to look at it as a possibility but wasn't sure if someone had already gone down this road.  I have played around with P0f but didn't see anything in documentation or the like.

Thanks

**Comment by weslambert** [comment, marked as answer]
I believe in the past P0f was used, but we don't currently have a dedicated tool for this.
